db/tikv/txn.go

```diff
@@ -153,43 +154,43 @@ func (db *txnDB) Scan(ctx context.Context, table string, startKey string, count
 }
 
 func (db *txnDB) Update(ctx context.Context, table string, key string, values map[string][]byte) error {
-	rowKey := db.getRowKey(table, key)
+	// Suppress RunInNewTxn warning log.
```

Comment:
Other than the Update operation, I'm not sure if other operations needs retry logic. I would like to get some feedback on whether we need to retry at all.